backend/molgenis-emx2/src/main/java/org/molgenis/emx2/User.java

```diff
@@ -0,0 +1,13 @@
+package org.molgenis.emx2;
+
+public class User {
```

Comment:
without equals user with username 'jan' != 'jan'

Reply:
yes, I was a bit in doubt wether to go that far. So I didn't. You think we should?

Reply:
added equals and hashcode

Reply:
all of the above and more is solved by records it seems